fl4health/clients/nnunet_client.py

```diff
@@ -170,7 +169,7 @@ def __init__(
         # nnunet specific attributes to be initialized in setup_client
         self.nnunet_trainer: nnUNetTrainer
         self.nnunet_config: NnunetConfig
-        self.plans: dict
+        self.plans: dict[str, Any]
```

Comment:
I'm not 100% sure, but I think we want to use the typing `Dict[str, Any]` vs. the python `dict[str, Any]`? Maybe they are identical or aliased, and this is unnecessary thought.

Reply:
They are aliases of one another. The typing versions of built-in types such as Dict are deprecated and will begin to generate warnings with python>3.14. See the docs [here](https://docs.python.org/3/library/typing.html#deprecated-aliases)